boot.py

```diff
@@ -1,3 +1,13 @@
+import sublime
+from .plugin import __version__
+
+if __version__ <= (0, 11, 0) and int(sublime.version()) > 4000:
```

Comment:
Why do we need to check the `__version__` at all?
Just checking `sublime.version()` seems sufficient.

Reply:
We don't need to check as long as we won't merge this change to ST4.
I'm OK with that if we can ensure that.

Reply:
> Why do we need to check the __version__ at all?

To make a difference between
the LSP version for ST3 and
the LSP version for ST4. 

> Just checking sublime.version() seems sufficient.

Not really. 
In that case, If we just check the Sublime version, we will get the error popup each time we open ST no meather the LSP version.

The intent is to show the error only for user who are using ST4 and have LSP version for ST3. 
Users who are using ST4 and the LSP version for ST4 should not/won't see the error message.

Did I understood you correctly @rchl?

Reply:
You understand correctly but I'm assuming we won't merge this change into ST4 branch and then it's fine.

Reply:
I got confused, I do want to get this into ST4 too. @predragnikolic is correct.
I guess it should say:

```python
if __version__ >= (0, 11, 0) and int(sublime.version()) < 4070:
```

Reply:
Then we should raise version of ST4 branch to be further away from ST3 branch.
Otherwise we can't handle that properly.
We could change ST4 branch version to 1.0.0 even. It doesn't really matter as long as they are separate and further apart.

Reply:
Both solutions are fine.
Either add tow different versions for LSP3 and LSP4 and do the LSP version check,

or.
Just check the sublime.version() (withouth the lsp version check)
merge it into master
merge master into st-4000
revert this commit on the st-4000

Reply:
Are there any pitfalls in maintaining two versions? It's boring maintaining even one version number.

Reply:
When we only do the check in master, and revert the commit in the st4 branch, then when we eventually merge the st4 branch into master, the version check is gone.

Reply:
I'm okay with it becoming an issue again once we merge st4 into master, so let's roll with this and undo the commit when master is merged into st4.

Reply:
So what to do then?
Add version check for LSP version? (Then we need to set them apart first).
Or just keep as is?
If maintaining two versions is gonna be a chore then I guess we should first come up with some script that helps with releasing (maybe based on gitgutter's).

Reply:
Re-read your last comment and as I understand we keep as is. :)